lib/Crypto/Util/asn1.py

```diff
@@ -687,6 +741,12 @@ def __init__(self, value=b'', implicit=None):
         """
         DerObject.__init__(self, 0x04, value, implicit, False)
```

Comment:
One thing I was considering was to move the "nested object" representation into here:

```diff
+        if isinstance(value, DerObject):
+            value = value.encode()
         DerObject.__init__(self, 0x04, value, implicit, False)
```

That would allow streamlining the `repr` code a good deal.